### PR TITLE
[Docs] synchronizes chinese version README wht English version

### LINKED ISSUE #1228: [suggestions] offer a Chinese verson of README
### Description

I suggest adding a Chinese version of README and OpenDigger's advantages and so on.

### PR DESCRIPTION
**Pull request by yanchaomei**
close #1228 This PR synchronizes Chinese README with English README

### CODE REVIEW

**Comment by frank-zsy** [comment]
Thanks for the PR, you should add `close #1228` in the PR body to make a relationship to the original issue and close it when the PR merged.

File: docs/zh-cn/README.md [hunk 1]
```diff
@@ -1,7 +1,298 @@
...
+您可随意使用这些数据来构建自己的数据应用程序，您可以将OpenDigger作为您的数据源，并且欢迎在您的项目中使用以下徽章来展示数据源。
+
+[![](https://img.shields.io/badge/Data-OpenDigger-2097FF)](https://github.com/X-lab2017/open-digger)
+
+### 对仓库而言
+
+<table>
+  <thead>
+    <tr>
+      <th>类型</th><th>名称</th><th>来源</th><th>例子</th><th>代码</th><th>CodePen</th>
```
Comment (frank-zsy) [inline docs/zh-cn/README.md:20]: The `index` should be 指标，the `metric` should be 度量 here。

File: docs/zh-cn/README.md [hunk 2]
```diff
@@ -1,7 +1,298 @@
...
+      <td><a href="https://oss.x-lab.info/open_digger/github/frank-zsy/repo_network.json">repo_network.json</a></td>
+      <td><a href="https://github.com/X-lab2017/open-digger/blob/master/src/cron/tasks/network_export.ts#L63">Link</a></td>
+      <td><a href="https://codepen.io/frank-zsy/pen/NWBVjpV?type=repo_network&name=frank-zsy">Demo</a></td>
+    </tr>
+  </tbody>
+</table>
+
+## 用户
+
+对于以上数据，OpenDigger 目前有一些用户。
```
Comment (frank-zsy) [inline docs/zh-cn/README.md:274]: 用户 can be 用户案例或应用场景。

**Comment by frank-zsy** [comment]
Thanks, LGTM

/approve

**Comment by zhicheng-ning** [comment]
It seems that some unexpected situation cause  merging is blocked ?

**Comment by frank-zsy** [comment]
@zhicheng-ning I think it is open-digger-bot's problem, I will update the bot later and merge this PR manually for now.

**Pull request merged**